<!-- -*- mode: XML -*- -->
<launch>

	
    	<arg name="speed" default='1' />
        <arg name="min_delta" default='-0.19' />
        <arg name="max_delta" default='0.19' />
        <arg name="delta_incr" default='0.04' />
        <arg name="dt" default='1' />
        <arg name="T" default='300' />
        <arg name="compute_time" default='0.09' />
        <arg name="laser_offset" default='1.0' />
    <node pkg="assignment3" type="laser_wanderer.py" name="laser_wanderer">
		<param name="speed" value="$(arg speed)" />
        <param name="min_delta" value="$(arg min_delta)" />
        <param name="max_delta" value="$(arg max_delta)" />
        <param name="delta_incr" value="$(arg delta_incr)" />
        <param name="dt" value="$(arg dt)" />
        <param name="T" value="$(arg T)" />
        <param name="compute_time" value="$(arg compute_time)" />
        <param name="laser_offset" value="$(arg laser_offset)" />
    </node>
  
</launch>
